en/snap/platform/overview.md

```diff
@@ -25,25 +25,27 @@ Step by step guide to integrate Snap to your platform of choice, will be explain
 
 ## Shopify
 
-Please complete the following steps:
+Please complete the steps given below:
 
 1. Create an online store with [Shopify](https://shopify.com).
-2. Register to Midtrans account [here](https://account.midtrans.com/register).
-3. Complete the account registration form, or get help by contacting [Midtrans activation team](mailto:activation@midtrans.com) with __SHOPIFY - URL Name__ as a subject header and mention your registered _Midtrans Merchant ID_.
+2. Register to [Midtrans account](https://account.midtrans.com/register).
+3. Complete the account registration form. Contact [Midtrans activation team](mailto:activation@midtrans.com) for help. Use **SHOPIFY - URL Name** as a subject header and mention your registered _Midtrans Merchant ID_.
 
-Note:
+?>***Note:***
 - You can try with Shopify Trial plan to test payment integration in Sandbox mode.
-- You may be required by Shopify to have an active paid-plan in order to allow your customer to do checkout on Production mode.
+- Shopify requires you to have an active paid-plan, to allow your customer to checkout on *Production* mode.
```

Comment:
> Shopify requires you

This is not always true, we never know if Shopify change policy etc. That is why it is intentional that we put words: `... may be required ...`, this is something we cannot control.